docs/configuration.md

```diff
@@ -94,6 +96,22 @@ The following variables are set.
 | OPENSTACK_CLOUD_PROVIDER_CONF_B64 | The content of [cloud.conf](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf) which is used by OpenStack cloud provider |
 | OPENSTACK_CLOUD_CACERT_B64 | (Optional) The content of your custom CA file which can be specified in your clouds.yaml by `ca-file` |
 
+[Application Credentials](https://docs.openstack.org/keystone/latest/user/application_credentials.html) is used for applications to authenticate to keystone. This way, the user’s password is not embedded in the application’s configuration, which is especially important for users whose identities are managed by an external system such as LDAP or a single-signon system.
+
+After your external cloud configuration (`/tmp/cloud.conf` by default) is ready, the `application-credential` related configuration is configured.
+
+```
+[Global]
+auth-url=http://x.x.x.x/identity
+domain-name="default"
+region="RegionOne"
+
+application-credential-id="xxxxxxxxxxx"
+application-credential-secret="xxxxxxxxxxxxxxxxx"
+```
```

Comment:
This is that we need to add application-credential-id and application-credential-secret after creating /tmp/cloud.conf.
That is not true. Please remove L101-111.